roles/inspec/defaults/main.yml

```diff
@@ -9,9 +9,9 @@ inspec:
   dependency_path: /etc/inspec/
   profiles:
     openstack_security:
-      name: inspec-openstack-security
+      name: openstack-baseline
```

Comment:
So this is not really creating a new profile. what remains on disc is a directory with inspec checks in it that are not invoked by inspec anymore. We can safely leave this directory in there.